### ISSUE TITLE
don't assume spacemacs

### ISSUE THREAD

**Issue by emacsomancer** [dalanicolai/pymupdf-mode.el#2]
lines 238-9 should check if the user is using spacemacs first

**Comment by dalanicolai** [comment]
Thanks, I am waiting for an update of https://github.com/dalanicolai/pymupdf-mode.el/pull/1. But otherwise I will fix it soon...

**Comment by jpggithub** [comment]
yes, it should be fixed because pymupdf can't be used with Emacs without spacemacs.
Thanks

**Comment by dalanicolai** [comment]
It should be fixed now, I think. Thanks for the reminder...
